packages/test-utils/src/ConditionalTest.ts

```diff
@@ -5,9 +5,11 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-/* eslint-disable jest/no-focused-tests */
-
 import semver = require('semver');
+import type {Global} from '@jest/types';
```

Comment:
Just a reminder. It is not possible to import from `@jest/globals`, because `@jest/globals` depend on `expect` which depends on `@jest/test-utils`. That’s circular. In contrary `@jest/types` currently depends only on `@jest/schemas`.

Reply:
We should fix the cycle - lerna warns every time I publish. Would be nice if yarn did the same on `install`

Reply:
Where does Lerna find a cycle?

In this comment I try to say that here importing from `@jest/types` does not create a cycle, but importing from `@jest/globals` would. In earlier PR it was your suggestion to import from `@jest/globals`. I tried that and hit the circular dips error. This is simply an explanation why I went for importing from `@jest/types` (;

Reply:
`pretty-format` has a dev dep on `expect`

Reply:
Right. I recall that. It was added in #12626

Reply:
Yup